Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/model/Tasks.java

```diff
@@ -107,4 +120,13 @@ public boolean apply(Map.Entry<Id, SyncedData<Task>> input) {
             }
         };
     }
+
+    @Override
+    public String toString() {
+        String result = "";
+        for (Id id : internalMap().keySet()) {
+            result = result + ", " + "{" + id + "; " + internalMap().get(id).syncState() + "}";
+        }
+        return result.substring(2);
```

Comment:
Fixed in https://github.com/novoda/spikes/pull/130/commits/9e4dbc3c496d02aedcb9de59d79690304f3f4bd5

I kept the delimiter string inside the `toString()` method - felt a bit cleaner not having it at the top of the class as a constant, because oonly the `toString()` method uses it.